client/src/document/on-github.tsx

```diff
@@ -118,7 +129,15 @@ function NewIssueOnGitHubLink({ doc }: { doc: Doc }) {
   if (!contentLabel) {
     contentLabel = "Other";
   }
-  sp.set("labels", `Content:${contentLabel},needs-triage`);
+  labels.push(`Content:${contentLabel}`);
+  if (locale !== "en-US") {
+    const localeLabel =
+      LOCALE_LABEL_ALIASES.get(locale.toLowerCase()) || locale.toLowerCase();
+    labels.push(`l10n-${localeLabel}`);
```

Comment:
what's lost with this approach is checking that there is a label for the said locale

Reply:
e.g. something about /ru/ would try to set a l10n-ru label which doesn't exist; I'm not sure how that affects the overall label-setting.

Reply:
That's actually fine. If you add `labels.push('garbage')` so that the URL has `&labels=needs-triage,garbage` GitHub will just ignore it.

Reply:
And for the record, I was worried about locales we don't have labels for like `l10n-de` or `l10n-es`. But first of all, my statement above means that GitHub would just ignore it. And second of all, when you view a `/de/docs/Foo/Bar` URL it doesn't even have the "Report a problem on GitHub" link in the footer. So we're in double-luck there :)